services/web-app/components/svg-libraries/action-bar/action-bar.module.scss

```
@@ -0,0 +1,109 @@
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/motion' as motion;
+@use '@carbon/react/scss/type' as type;
+
+@media screen and (prefers-reduced-motion: reduce) {
+  .container {
+    position: absolute;
+    z-index: 1;
+    right: 0;
+    bottom: 0;
+    display: flex;
+    opacity: 1;
+    transition: none;
+
+    svg {
+      fill: theme.$icon-primary;
+    }
+  }
+}
```

```suggestion
@media screen and (prefers-reduced-motion: reduce) {
  .container {
    transition: none;
  }
}
```

I think we can simplify to just this?